cmd/crc/cmd/daemon.go

```diff
@@ -20,6 +38,106 @@ var daemonCmd = &cobra.Command{
 		logging.CloseLogging()
 		logging.InitLogrus(logging.LogLevel, constants.DaemonLogFilePath)
 
-		runDaemon()
+		go runDaemon()
+		go customDNS()
+
+		apiSock := filepath.Join(constants.CrcBaseDir, "network.sock")
+		_ = os.Remove(apiSock)
+		endpoints := []string{
+			transport.DefaultURL,
+			fmt.Sprintf("unix://%s", apiSock),
+		}
+
+		if err := run(&types.Configuration{
+			Debug:             isDebugLog(),
+			CaptureFile:       captureFile(),
+			MTU:               4000,
+			Subnet:            "192.168.127.0/24",
+			GatewayIP:         "192.168.127.1",
+			GatewayMacAddress: "\x5A\x94\xEF\xE4\x0C\xDD",
+			DNSRecords: map[string]net.IP{
+				"gateway.crc.testing.":              net.ParseIP("192.168.127.1"),
+				"apps-crc.testing.":                 net.ParseIP("192.168.127.2"),
+				"etcd-0.crc.testing.":               net.ParseIP("192.168.127.2"),
+				"api.crc.testing.":                  net.ParseIP("192.168.127.2"),
+				"api-int.crc.testing.":              net.ParseIP("192.168.127.2"),
+				"oauth-openshift.apps-crc.testing.": net.ParseIP("192.168.127.2"),
+				"crc-zqfk6-master-0.crc.testing.":   net.ParseIP("192.168.126.11"),
```

Comment:
instead of hard code those domains we should take it from bundle metadata side.

Reply:
`crc-zqfk6-master-0` is not consustent, metadata.
however these other values not defined in the SNC generation step.

I do think a more centralized approach to the IP settings is necessary

Reply:
In fact, I tested with multiples bundles and this record is not used at all.  So we don't depend on the master node name apparently.

Reply:
the name might internally be recorded somehow... so, why not test this without? this way we can confirm if this is a necessity or not. Things might have changed over time.

Reply:
I retest it multiples times and I can see DNS requests for the `crc-.*-master-0.crc.testing.` domain but even if it doesn't work, the cluster seems to work fine. 🤔

Reply:
> It remains the hardcoded domain .crc.testing. I guess we can keep it as it is for the moment. It won't change in a near future.

This hardcoded domain seems like it could be avoided with a `crc route add` command, it could be called early by `machine.Start()`  once it knows which domains the cluster needs.